example/README.md

```diff
@@ -0,0 +1,36 @@
+# Example of ELK stack with Filebeat for collecting stats from LanCache
+
+This is an example set up of the ELK stack with a filebeat instance for reading the logs from LanCache. This should be a very quick way to get a working monitoring stack however should not be used in production!
+
+Please make sure to change the password of the default elastic user, this can done by searching for "changeme" and replacing. 
+
+This assumes you already have a running LanCache setup, for instructions on how to do that please see the [LanCache website](https://lancache.net/docs).
+
+## Quickstart
+
+1. Clone this repository
+2. Change the password of elasticsearch by search and replacing "changeme" in the repository.
+3. Modify the logstash.conf outputs like so
+```
+output {
+  elasticsearch {
+    hosts    => [ 'elasticsearch' ]
+    user => 'elastic'
+    password => 'changeme'
+    ssl => false
+    index => "%{my_index}"
+  }
+}
+```
+Ensure you change the "changeme" here to your new password.
+4. Execute docker-compose up -d in the example directory
+5. Visit hostname:5601 (hostname will be localhost if running locally) and login with elastic and the password you changed.
+6. Go to Stack Management via the left sidebar, go to Saved Objects
```

Comment:
i assume there's no way to api this?

Seems to imply this might work?
https://www.elastic.co/guide/en/kibana/master/saved-objects-api-import.html

Reply:
This would require a separate one-shot container, which I wasn't a big fan of personally, though can be implemented easily.

Reply:
if it works i think it's more useful with a oneshot. Experience tells me most people will forgot to run or break it!

Reply:
Will have a dig into this 👍

Reply:
We've run into a few issues with the currently exported dashboards so I think we're going to fix them up and then re-export into the new NDJSON format and provide them as well.

Reply:
@ewancolyer thanks